main.py

```diff
@@ -10,15 +10,19 @@
 from translate import Translator
 
 from utils import Cooldown, TopicGenerator, bot_dir, console, send_message
+from configparser import ConfigParser
 
 __version__ = "0.6.0"
 
 load_dotenv(
     dotenv_path=bot_dir / ".env"
 )  # Added path to support older versions of python-dotenv
 
-tell_me_to_cooldown = Cooldown(200)
-topic_cooldown = Cooldown(100)
+config = ConfigParser()
+config.read("brainbot.ini")
+
+tell_me_to_cooldown = Cooldown(config.getint("cooldowns", "tell_me_tos", fallback=200))
```

Comment:
```suggestion
tell_me_to_cooldown = Cooldown(config.getint("cooldowns", "tell_me_to", fallback=200))
```
Just this little typo and everything else looks good to me!

Reply:
ah yeah, my bad. fixed it 😅